etabotsite/etabotsite/settings.py

```diff
@@ -266,15 +267,7 @@
 
 SYS_DOMAIN = local_host_url if LOCAL_MODE else prod_host_url
 
-sys_email_settings = {}
-try:
-    with open('sys_email_settings.json') as f:
-        sys_email_settings = json.load(f)
-    logging.info('loaded sys_email_settings.json: {}'.format(
-        sys_email_settings.keys()))
-except Exception as e:
-    logging.warning('Cannot load sys_email_settings due to "{}". \
-Will use default values'.format(e))
+sys_email_settings = custom_settings.get('SYS_EMAIL_SETTINGS')
```

Comment:
what would happen if custom_settings doesn't have this key?
please accommodate for this use case

Reply:
Added if to check if the key existed in the custom settings, if not we print a warning. If it is in the custom settings we load all the information. 

Also added a check for the AUTHLIB to spit out warning if there is an issue.